CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
+### Changed
+- added in colour variables for discovery and live badges and banners
+- updated link colour variables to match gov.uk [#795](https://github.com/hmrc/assets-frontend/pull/795)
```

Comment:
No, - added in colour variables for discovery and live badges and banners - highlights the introduction of new variables
and - updated link colour variables to match gov.uk - highlights existing variables have been updated.

Will capitalise them now.